pkg/clusterconfig/clusterconfig.go

```diff
@@ -68,47 +66,23 @@ func GetCoreClient() (*coreset.CoreV1Client, error) {
 	return client, nil
 }
 
-func GetInstallConfig() (*installer.InstallConfig, error) {
-	client, err := GetCoreClient()
-	if err != nil {
-		return nil, err
-	}
-
-	cm, err := client.ConfigMaps(installerConfigNamespace).Get(installerConfigName, metav1.GetOptions{})
-	if err != nil {
-		return nil, fmt.Errorf("unable to read cluster install configuration: %v", err)
-	}
-
-	installConfig := &installer.InstallConfig{}
-	if err := yaml.NewYAMLOrJSONDecoder(strings.NewReader(cm.Data["install-config"]), 100).Decode(installConfig); err != nil {
-		return nil, fmt.Errorf("unable to decode cluster install configuration: %v", err)
-	}
-
-	return installConfig, nil
-}
-
-func GetAWSConfig(listers *regopclient.Listers) (*Config, error) {
+func GetAWSConfig(ctx context.Context, client rest.Interface, listers *regopclient.Listers) (*Config, error) {
 	cfg := &Config{}
 
-	installConfig, err := GetInstallConfig()
+	infra, err := osclientset.New(client).ConfigV1().Infrastructures().Get("cluster", metav1.GetOptions{})
 	if err != nil {
 		return nil, err
 	}
 
-	if installConfig.Platform.AWS != nil {
-		cfg.Storage.S3.Region = installConfig.Platform.AWS.Region
-	}
-
-	client, err := GetCoreClient()
-	if err != nil {
-		return nil, err
+	if infra.Status.Platform == configv1.AWSPlatformType {
+		cfg.Storage.S3.Region = "us-east-1" // FIXME: installConfig.Platform.AWS.Region
```

Comment:
I'm not sure what to do about this.  Until we get it sorted:

/hold

Reply:
It's possible the region is part of the cloud-provider config (which is availble via `Infrastructure` since openshift/api#245).

Reply:
@abhinavdahiya suggests getting this from the metadata service (and using the region where your node is running), although I don't have example code at hand for that yet.